### DISCUSSION TITLE
Gatt echo example, no services found

### DISCUSSION THREAD

**Discussion by legounix** [bluez/bluer#83] [General]
Hey, I'm currently tring out the gatt echo server/client example, but somehow the `services()`call returns an empty vec and therefore never retrieves the provided characteristics. I have no clue how to debug this.

I'm on latest main branch and bluez is version 5.66-1

**Comment by paulirotta** [comment]
There are some binaries in bluer-tools which may help you get a working setup to help cross-check.
